build.gradle

```diff
@@ -52,7 +52,7 @@ repositories {
 }
 
 dependencies {
-    compile "mcp.mobius.waila:Waila:${waila_version}"
+    compile "mcp.mobius.waila:Waila:${waila_version}:dev"
```

Comment:
Deobf jars are deprecated and most mods don't even provide them in 1.8 (didn't realize WAILA did). Use deobfCompile as the target.